.github/workflows/lint.yml

```
@@ -18,12 +18,16 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v4.0.2
         with:
-          node-version: 21.x
+          node-version: 22.x
+
       - name: Install Yarn
         run: npm install -g yarn
+
       - name: Install dependencies
         run: yarn install --immutable
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)